cita-executor/core/src/libexecutor/command.rs

```diff
@@ -283,6 +282,7 @@ impl Commander for Executor {
             block_quota_limit: *header.quota_limit(),
             account_quota_limit: u64::max_value().into(),
         };
+        context.block_quota_limit = U256::from(self.sys_config.block_quota_limit);
```

Comment:
It's so weird. Why not just set the block_quota_limit at the first.

Reply:
The `block_quota_limit` in context from header is a default value. It should be updated. at first?